python_scripts/cross_validation_time.py

```diff
@@ -220,5 +220,5 @@
 # %% [markdown]
 # In conclusion, it is really important to not use an out of the shelves
```

Comment:
I don't think "out of the shelves" is grammatically correct nor a common term for the students to know. Maybe we can rephrase the whole paragraph.